daprdocs/content/en/reference/api/configuration_api.md

```diff
@@ -0,0 +1,87 @@
+---
+type: docs
+title: "Configuration API reference"
+linkTitle: "Configuration API"
+description: "Detailed documentation on the configuration API"
+weight: 700
+---
+
+## Component file
+
+A Dapr `configuration.yaml` component file has the following structure:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: configuration.<TYPE>
+  version: v1
+  metadata:
+  - name:<KEY>
+    value:<VALUE>
+```
+
+| Setting | Description                                                                            |
+| ------- |----------------------------------------------------------------------------------------|
+| `metadata.name` | The name of the configuration store.                                                   |
+| `spec/metadata` | An open key value pair metadata that allows a binding to define connection properties. |
+
+## Get Configuration
+
+This endpoint lets you get configuration from store.
+
+### HTTP Request
+
+```
+POST http://localhost:<daprPort>/v1.0/configuration/<storename>
+```
+
+#### URL Parameters
+
+Parameter | Description
+--------- | -----------
+`daprPort` | The Dapr port
+`storename` | The `metadata.name` field in the user-configured `configuration.yaml` component file. Refer to the [Dapr configuration store component file structure](#component-file) mentioned above.
+
+The optional request, like configuration key and metadata are passed via URL query parameters. For example:
+```
+GET http://localhost:3500/v1.0/configuration/myConfigurationStore?key=myConfigKey&metadata.contentType=application/json
```

Comment:
I would have expected this to have a key value in the URL and not in the query parameters, 
to be consistent with the state management GET API. Like this?

GET http://localhost:3500/v1.0/configuration/myConfigurationStore/myConfigKey